indexer/drawing_rule_def.hpp

```diff
@@ -2,9 +2,21 @@
 
 #include "base/buffer_vector.hpp"
 
+#include "drape/utils/projection.hpp"
 
 namespace drule
 {
+
+// Priority range for area and line drules. Should be same as PRIORITY_RANGE in kothic.
+// Each layer = +/-1 value shifts the range by this number, so that e.g. priorities
+// of the default layer=0 range [0;1000) don't intersect with layer=-1 range [-1000;0) and so on..
+double constexpr kLayerDepthRange = 1000;
+
+// Should be same as OVERLAYS_MAX_PRIORITY in kothic.
+int32_t constexpr kOverlaysMaxPriority = 10000;
+// Negative range is used for optional captions which are below all other overlays.
+static_assert(dp::kMinDepth <= -kOverlaysMaxPriority && kOverlaysMaxPriority <= dp::kMaxDepth);
```

Comment:
Move this constrain into drape code. Say where kMinDepth is declared.

Reply:
possible, but it'll be controversial why an overlays constant is in ogl projection settings

Reply:
You can place this static_assert wherever in drape project.

Reply:
I think I found a better place for it..